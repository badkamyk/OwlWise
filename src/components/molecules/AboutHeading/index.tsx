import HeadingMain from "../../atoms/HeadingMain";
import MainContentParagraph from "../../atoms/MainContentParagraph";

export default function AboutHeading() {
	return (
		<div className="py-8 px-4 text-center mx-auto max-w-screen-xl lg:py-16 lg:px-6 ">
			<HeadingMain firstPhrase={"About"} secondPhrase={"us"} />
			<MainContentParagraph color={"white"} size={"xl"}
														text={"We are a team of software engineers who are passionate about building scalable solutions for businesses."} />
		</div>
	);
}