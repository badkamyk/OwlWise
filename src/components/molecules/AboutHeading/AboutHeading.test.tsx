import { render, screen } from "@testing-library/react";
import { expect, it } from "vitest";
import AboutHeading from "./index";

it("AboutHeading", () => {
	const { container } = render(<AboutHeading />);
	expect(screen.getByText("About")).toBeTruthy();
	expect(screen.getByText("us")).toBeTruthy();
	expect(
		screen.getByText(
			"We are a team of software engineers who are passionate about building scalable solutions for businesses."
		)
	).toBeTruthy();
	expect(container).toMatchSnapshot();
});
