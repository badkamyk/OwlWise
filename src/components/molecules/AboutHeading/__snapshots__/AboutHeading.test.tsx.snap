// Vitest Snapshot v1

exports[`AboutHeading 1`] = `
<div>
  <div
    class="mx-auto max-w-screen-xl py-8 px-4 text-center lg:py-16 lg:px-6 "
  >
    <h2
      class="mb-3 text-3xl font-extrabold text-gray-900 dark:text-white md:text-5xl lg:text-6xl"
      data-testid="heading-main"
    >
      <span
        class="bg-gradient-to-r from-sky-400 to-emerald-600 bg-clip-text text-transparent"
      >
        About
      </span>
       
      us
    </h2>
    <p
      class="mb-4 text-white text-xl"
      data-testid="main-content-paragraph"
    >
      We are a team of software engineers who are passionate about building scalable solutions for businesses.
    </p>
  </div>
</div>
`;
